docs/tutorial/updates.md

```diff
@@ -0,0 +1,44 @@
+# Updating Applications
+
+There are several ways to update an Electron application. The easiest and officially supported one is taking advantage of the built-in [Squirrel](https://github.com/Squirrel) framework and the [autoUpdater](../api/auto-updater.md) module that comes with it.
+
+## Deploying an Update Server
+
+To get started, you firstly need to deploy an update server (that's where the [autoUpdater](../api/auto-updater.md) module will download new updates from).
+
+Depending on your needs, you can choose from one of these:
+
+- [Hazel](https://github.com/zeit/hazel) – Pulls new releases from [GitHub Releases](https://help.github.com/articles/creating-releases/) and is **perfect for getting started**
+- [Nuts](https://github.com/GitbookIO/nuts) – Also uses [GitHub Releases](https://help.github.com/articles/creating-releases/), but caches app updates on disk
+- [electron-release-server](https://github.com/ArekSredzki/electron-release-server) – Provides you with a dashboard for handling releases
+
+## Implementing Updates into Your App
+
+Once you've deployed your update server, continue with importing the required modules in your code (the following code might vary for different server software, but it works like described when using [Hazel](https://github.com/zeit/hazel)).
+
+**Important:** Please ensure that the code below will only be executed in production - you can use [electron-is-dev](https://github.com/sindresorhus/electron-is-dev) to check for the environment).
+
+```js
+const { app, autoUpdater } = require('electron')
+```
+
+Next, put together the URL of the update server:
+
+```js
+const server = <your-deployment-url>
+const feed = `${server}/update/${process.platform}/${app.getVersion()}`
+```
+
+As the final step, tell [autoUpdater](../api/auto-updater.md) where to ask for updates:
+
+```js
+autoUpdater.setFeedURL(feed)
```

Comment:
Maybe add `autoUpdater.checkForUpdates()` here as well or people might think it does the checking automagically